pandas/core/frame.py

```diff
@@ -201,6 +201,8 @@
     nargsort,
 )
 
+from pandas.api.exchange.dataframe_protocol import DataFrame as DataFrameXchg
```

Comment:
as a rule implementations belong in core (or io/plotting/whatever) and we import those into pd.api, not the other way around

Reply:
this particular line would also go in the `if TYPE_CHECKING:` block